packages/fuel-gauge/test/fixtures/forc-projects/bytecode-sway-lib/src/main.sw

```diff
@@ -19,6 +19,6 @@ impl MyContract for Contract {
     }
 
     fn compute_predicate_address(bytecode: Vec<u8>) -> Address {
-        return Address::from(0x6b6ef590390f0a7de75f8275ab5d7877c17236caba2514039c6565ec15f79111);
+        return Address::from(0x68fec7a57e48f4ec6467d7e09c27272bd8ca72b312ea553a470b98731475ccf3);
```

Comment:
Why did this address change?

Reply:
https://github.com/FuelLabs/fuels-ts/blob/c40a421d88f12050fca3f63a1b917b3751d77bc4/packages/fuel-gauge/src/bytecode-sway-lib.test.ts#L60-L66

We now pass through an actual Predicate, rather than the hardcoded files.